pymongo/asynchronous/collection.py

```diff
@@ -1176,8 +1188,12 @@ async def replace_one(
             aggregate expression context (e.g. "$$var").
         :param comment: A user-provided comment to attach to this
             command.
+        :param sort: Specify which document the operation updates if the query matches
+            multiple documents. The first document matched by the sort order will be updated.
```

Comment:
Here and elsewhere add: Requires MongoDB >=8.0.

Reply:
Done